### ISSUE TITLE
ReasonReact and JSX support

### ISSUE THREAD

**Issue by okonet** [reasonml-editor/reasonml-idea-plugin#31]
Any plans to support ReasonReact and JSX in particular in the near future?

**Comment by giraud** [comment]
Totally (well, it depends on what you mean by 'near' :) )

Do you have a more detailed request ?

**Comment by okonet** [comment]
No, was more interested if I should switch to another editor for the time being or is this planned for near future :)

**Comment by giraud** [comment]
is 'support' an equivalent for 'completion', or is it more than this ?

**Comment by okonet** [comment]
I'd be happy about the code highlighting already. If the completion will be available, it would be 🔥

**Comment by giraud** [comment]
v0.38 should have a better parsing and highlighting of JSX tags

**Comment by okonet** [comment]
Thanks! I’ll give it a try!

**Comment by giraud** [comment]
I'm closing that issue. please fill more detailed issues for next bugs, thanks
